src/lib/uptime_service/uptime_service.ml

```diff
@@ -16,6 +16,7 @@ type block_data =
   ; peer_id : string
   ; snark_work : string option [@default None]
   ; graphql_control_port : int option [@default None]
+  ; block_producer_version : string option [@default None]
```

Comment:
Why is this called `block_producer_version`? It's the version of the node, regardless if the node is a block producer or not. Also, it's not actually the version, it's the commit we built from (which is a separate concept from versioning). I would recommend just calling this `commit_sha` or `built_with_commit_sha`.

Reply:
It's called that because it's the use case for that field, to identify the "version of the code" of the BP node that sent the payload. But yes, those names are more semantically accurate.